docs/references/vod/upload.mdx

```diff
@@ -60,12 +110,38 @@ curl --location --request POST 'https://livepeer.studio/api/asset/request-upload
 Now upload (`PUT`) a video file on the exact URL in the `url` field of the
 response above:
 
-```jsx
+
+
+<Tabs>
+  <TabItem value="bash" label="bash">
+
+```bash
 curl --location --request PUT "${url}" \
 --header 'Content-Type: video/mp4' \
 --data-binary '@$VIDEO_FILE_PATH'
 ```
 
+</TabItem>
+
+<TabItem value="py" label="Fetch">
+
+```js
+ fetch(`${assetURL}`, {
+        method: "PUT",
+        headers: {
+          "Content-Type": "video/mp4",
+        },
+        body: file, // Asset for upload
```

Comment:
This comment is confusing, as asset actually refers to the API resource, not to the content of it.

```suggestion
        body: file, // asset content to upload, grab one from file system libs
```